.changeset/beige-crabs-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-tools/code-file-loader': major
+---
+
+Fix Code File Loader's Bug
```

Comment:
This will go to the change logs so can we be more specific here? Thanks! 😊

Reply:
Sure! I add some new stuff, plz help me review it ~~ thanks